### DISCUSSION TITLE
Bug with Playerwarps and Claims

### DISCUSSION THREAD

**Discussion by HavocStudio** [GriefPrevention/GriefPrevention#1719] [Ideas]
![2021-12-19_18 21 02](https://user-images.githubusercontent.com/96396732/146694670-73c281cd-78f0-4ba1-8965-6518b631b119.png)
![2021-12-19_18 16 48](https://user-images.githubusercontent.com/96396732/146694671-81ad44f4-9f82-42b5-a961-9bd5f90b9bd8.png)

Player are able to set player warps in players claims when they are on and anvil showed above.

**Comment by bigpresh** [comment]
Not at all familiar with PlayerWarps, but it would need to be firing an event that GP would be looking for and able to prevent in claims (e.g. a `PlayerInteractEvent` or similar) and know to check if it was cancelled too.

Is this something that used to work correctly (i.e. PlayerWarps warps not being possible to create within someone else's claim) and stopped working with a new GP build / new PlayerWarps build / with 1.18.1, or is it a feature request that you'd like it to work but it never has so far?  More details would be helpful.
